src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexpsinglelinejava/Example3.java

```diff
@@ -0,0 +1,51 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="RegexpSinglelineJava">
+      <property name="format" value="Example3"/>
```

Comment:
Please make value to be "debug".
And rename some variable in code to be this name

Reply:
Done